### ISSUE TITLE
Local testing fails on WSL 

### ISSUE THREAD

**Issue by MiraArndt** [pep-dortmund/homepage#498]
When trying to set up local testing as suggested in the README.md, I get the following error:
`Liquid Exception: invalid byte sequence in UTF-8 in /mnt/c/Users/MiraArndt/Documents/PeP/homepage/_posts/2015/2015-10-20-bachelorkolloqu.md`
I'm using WSL 1 running Ubuntu 22.04.1LTS on windows 10. 
```
$ ruby --version
ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x86_64-linux-gnu]
```
```
$ bundler --version
Bundler version 2.3.9
```

Also a file named `''$'\001''�'` gets magically created after running bundler.

**Comment by maxnoe** [comment]
Can't reproduce in WSL running Ubuntu 22.04 and the same versions of bundler and ruby. 

Is that on current main or in a branch you made?

Checking with python, the file is valid UTF-8:
´´´
open('./_posts/2015/2015-10-20-bachelorkolloqu.md').read()
´´´

From the path in the error message I gather that you are working in a Windows directory. Can you test if this works in a directory inside the WSL. E.g. just in the home? 

```
$ cd 
$ git clone ...
...
```

**Comment by maxnoe** [comment]
Also can't reproduce on a windows path

**Comment by MiraArndt** [comment]
Also doesn't work in the home directory. It's the current main, I didn't make any changes

**Comment by maxnoe** [comment]
We narrowed it down to the katex_block plugin we have... Removing all posts that use the `{% latex %}` block fixes the issue....

**Comment by maxnoe** [comment]
Should be fixed by #512, can you confirm @MiraArndt ?

**Comment by MiraArndt** [comment]
yes, it works now for me. I can build the website
